flytekit/clients/friendly.py

```diff
@@ -29,7 +28,7 @@
 from flytekit.models.core import identifier as _identifier
 
 
-class SynchronousFlyteClient(_RawSynchronousFlyteClient):
+class SynchronousFlyteClient(flyteidl.RawSynchronousFlyteClient):
```

Comment:
The key modification

Reply:
the other key modification needed is remove these two lines https://github.com/flyteorg/flytekit/blob/a7285067bfd90d53c373163c7db961fe09e3231d/pyproject.toml#L27-L28

Reply:
Yep sure of course, not only "grpcio", "grpcio-status", but also "flyteidl"